skema/img2mml/api.py

```diff
@@ -1,31 +1,45 @@
 import json
 import os
-from pathlib import Path
 import requests
-import re
-from skema.rest.proxies import SKEMA_MATHJAX_ADDRESS
+from pathlib import Path
+
+# from skema.rest.proxies import SKEMA_MATHJAX_ADDRESS
```

Comment:
You reference this later in your file, but you've commented out the import.

Reply:
@myedibleenso @ualiangzhang  - there was a small test here: https://github.com/ml4ai/skema/blob/main/skema/img2mml/tests/test_im2mml.py

But I don't know if it works anymore with @ualiangzhang 's changes. Also, it doesn't test the REST API, just the library function.

Reply:
Thanks for clarification, @adarshp! I have been actually editing this file and reorganizing the code to test more.